.storybook/stories/Text.stories.tsx

```diff
@@ -107,11 +107,6 @@ function TextShadowScene() {
       font="https://fonts.gstatic.com/s/raleway/v14/1Ptrg8zYS_SKggPNwK4vaqI.woff"
       anchorX="center"
       anchorY="middle"
-      outlineOffsetX={'10%'}
```

Comment:
Is there a reason you removed these?

Reply:
Sorry, my bad this slipped through ... no actually there is no reason. I'll fix that.

Reply:
Should be fixed now by [e7af40b](https://github.com/pmndrs/drei/pull/372/commits/e7af40b43b635319319ccf0b4dbefc2fc63b11e8)